setup_scripts/setup.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+function inst {
+    local typ="$1" # Get the type of install
+    shift;
+    for p in $@
+    do
+        if [ "$typ" == 'yum' ]; then
+            echo "yum install $p"
+            sudo yum -y install $p
```

Comment:
`sudo` internal to script generally considered bad practice. Looking at the setup of this, I'll argue that it's reasonable to expect that this will be run in an admin context, be that in su mode or with sudo is irrelevant.